include/seqan3/argument_parser/detail/format_help.hpp

```diff
@@ -17,11 +17,10 @@
 #include <cassert>
 #include <iostream>
 
-#include <range/v3/view/repeat_n.hpp>
-
 #include <seqan3/argument_parser/detail/format_base.hpp>
 #include <seqan3/core/detail/debug_stream_type.hpp>
 #include <seqan3/core/detail/terminal.hpp>
+#include <seqan3/range/views/repeat_n.hpp>
```

Comment:
Kinda unrelated :D

Reply:
well, not really. Because it some how broke due to removing other stuff. So there must have been a weird include graph. I only fixed stuff that was broken due to the changes.